configuration/coding_plans.py

```diff
@@ -487,6 +487,87 @@ def get_rqa_coding_plans(pipeline_name):
                    raw_field_fold_strategy=FoldStrategies.concatenate),
 ]
 
+S02_RQA_CODING_PLAN = [
+    CodingPlan(raw_field="covid19_promo_s02e01_raw",
+               time_field="sent_on",
+               run_id_field="covid19_promo_s02e01_run_id",
+               coda_filename="IMAQAL_COVID19_promo_s02e01.json",
+               icr_filename="covid19_promo_s02e01.csv",
+               coding_configurations=[
+                   CodingConfiguration(
+                       coding_mode=CodingModes.MULTIPLE,
+                       code_scheme=CodeSchemes.S02E01_PROMO,
```

Comment:
If the field name is promo_s02e01 then the code scheme should probably be called PROMO_S02E01

Reply:
updated thanks.